content/articles/add_docs.md

```diff
@@ -13,31 +13,40 @@ This is a good opportunity for you to contribute something to the community with
 
 ## Fork the repo!
 
-Our site is a static site hosted somewhere out there on the Internet. Super mysterious. You can find the repository of our site here: [https://github.com/SADevs/sadevs.github.io](https://github.com/SADevs/sadevs.github.io). Fork this repository and clone it to your local machine. The site is powered by Pelican, which is a static site generator written in Python. If you have Python on your machine and you want to see what your article looks like on your local machine, install Pelican by typing in your terminal `pip install pelican Markdown`. This will get the necessary packages to use this static site generator.
+Our site is a static site hosted somewhere out there on the Internet. Super mysterious. You can find the repository of our site here: [https://github.com/SADevs/sadevs.github.io](https://github.com/SADevs/sadevs.github.io). Fork this repository and clone it to your local machine. The site is powered by Pelican, which is a static site generator written in Python. If you have Python on your machine and you want to see what your article looks like on your local machine, install Pelican by typing in your terminal 
 
-If you don't care about what your article looks like on the site and just want to write content, copy this template to get you started into a new file with a relevant name of your article and ending with a markdown extension:
+    pip install pipenv # if you don't have pipenv on your machine already
+    pipenv install pelican Markdown
```

Comment:
no, it should just be `pipenv install`

It will pull the dependencies from the pipfile